### DISCUSSION TITLE
No faces?

### DISCUSSION THREAD

**Discussion by gerroon** [bpatrik/pigallery2#622] [Q&A] [answered]
Hi

I am using it on Debian testing (x64) using  the lines below. When I click on the faces tab I get "no faces to show" Is this not included in the docker setup?

```
docker run \
   -p 80:80 \
   -e NODE_ENV=production \
   -v <path to your config file folder>:/app/data/config \
   -v <path to your db file folder>:/app/data/db \
   -v <path to your images folder>:/app/data/images \
   -v <path to your temp folder>:/app/data/tmp \
   bpatrik/pigallery2:latest

```

**Comment by bpatrik** [comment, marked as answer]
App only reads faces from iptc/exif. Cannot recognize them on its own. Use an external tool for face tagging.

Also you need to index you gallery for the faces to show up.
